docs/tutorials/blog.md

```diff
@@ -84,28 +84,27 @@ You can put it anywhere you like. I stuck it right above the icons of all the te
 
 ![Screenshot of the app showing the blog post link][screenshot-of-the-app-showing-the-blog-post-link]
 
-<docs-info>You may have noticed we're using <a href="https://tailwindcss.com">tailwind</a> classes.</docs-info>
+<docs-info>You may have noticed we're using <a href="https://tailwindcss.com">Tailwind CSS</a> classes.</docs-info>
 
-The Remix Indie stack has [tailwind][tailwind] support pre-configured. If you'd prefer to not use tailwind, you're welcome to remove it and use something else. Learn more about your styling options with Remix in [the styling guide][the-styling-guide].
+The Remix Indie stack has [Tailwind CSS][tailwind] support pre-configured. If you'd prefer to not use Tailwind CSS, you're welcome to remove it and use something else. Learn more about your styling options with Remix in [the styling guide][the-styling-guide].
 
 Back in the browser go ahead and click the link. You should see a 404 page since we've not created this route yet. Let's create the route now:
 
-💿 Create a new file in `app/routes/posts/index.tsx`
+💿 Create a new file at `app/routes/posts._index.tsx`
```

Comment:
I just tried with v2_routeConvention: true and it seems to be working with /routes/posts.index.tsx (instead of _index.tsx)